### PR TITLE
Format amount on cancel order notification (ED-204)

### PR DESCRIPTION
**Pull request by kkostalkowicz**

### CODE REVIEW

File: src/blockchain/network.ts [hunk 1]
```diff
@@ -108,20 +108,20 @@ export type GasPrice$ = Observable<BigNumber>;
...
-          Accept: 'application/json',
-        },
-        crossDomain: true,
-      })),
-      retryWhen(errors => errors.pipe(delayWhen(() => onEveryBlock$.pipe(skip(1))))),
-      map(({ response }) =>
-        new BigNumber(response.average).times(1.1).times(new BigNumber(10).pow(8))
-      ),
-    )
+    // onEveryBlock$.pipe(
```
Comment (peculiarity) [inline src/blockchain/network.ts]: Can we just remove it ?

File: src/Main.tsx [hunk 2]
```diff
@@ -1,25 +1,41 @@
...
+      <theAppContext.Provider value={setupAppContext()}>
+        <Router history={browserHistoryInstance}>
+          <MainContentWithRouter />
+        </Router>
+      </theAppContext.Provider>
+    );
+  }
+}
+
+interface RouterProps extends RouteComponentProps<any>{}
```
Comment (peculiarity) [inline src/Main.tsx]: If there are no additional props why to create an "empty" interface? I guess if we need in the future to extend the standard `RouteComponentProps` then we can add that interface

File: src/exchange/offerMake/OfferMakeForm.stories.tsx [hunk 3]
```diff
@@ -305,25 +308,26 @@ stories.add('Validations on form', () => {
...
     </div>
+    </BrowserRouter>
   );
 });
 
 stories.add('Three validations error on one element', () => {
   const messages: Message[] = [
     {
-      kind: MessageKind.noAllowance,
+      kind: MessageKind.insufficientAmount,
```
Comment (peculiarity) [inline src/exchange/offerMake/OfferMakeForm.stories.tsx]: Probably I"m missing the idea but why did you change that?

File: src/blockchain/network.ts (hunk 1 above)
Comment (kkostalkowicz) [reply src/blockchain/network.ts]: This is temporary on purpose. After testing we either remove the comment or revert

File: src/Main.tsx (hunk 2 above)
Comment (kkostalkowicz) [reply src/Main.tsx]: The convention is to name the Props type, which can either extend an interface, contain some fields, or both extend and add fields. The app has other instances of Props that extends but adds no fields

File: src/exchange/offerMake/OfferMakeForm.stories.tsx (hunk 3 above)
Comment (kkostalkowicz) [reply src/exchange/offerMake/OfferMakeForm.stories.tsx]: I could alternatively wrap the form with a Router, but decided only to wrap the story which displays all possible errors, and skip other stories.
